pkg/server/options/batteries/batteries.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	// WorkspaceTypes leads to creation of a number of default types beyond the universal type.
-	WorkspaceTypes = "cluster-workspace-types"
+	WorkspaceTypes = "workspace-types"
```

Comment:
Don't we want to move the `RootComputeWorkspace` battery definition to the TMC side ?

Reply:
follow-up. This is just the first step. There must be many more.

Reply:
OK, sure